tasks/eks.py

```diff
@@ -0,0 +1,87 @@
+import os
+from invoke import task
+import yaml
+from .destroy import destroy
+from .deploy import deploy
+from . import doc
+from typing import Optional
+from invoke.context import Context
+from . import tool
+import pyperclip
+
+scenario_name = "aws/eks"
+
+
+@task(
+    help={
+        "install_agent": doc.install_agent,
+        "agent_version": doc.container_agent_version,
+        "stack_name": doc.stack_name,
+        "eks_linux_node_group": doc.linux_node_group,
+        "eks_linux_arm_node_group": doc.linux_arm_node_group,
+        "eks_bottlerocket_node_group": doc.bottlerocket_node_group,
+        "eks_windows_node_group": doc.windows_node_group,
+    }
+)
+def create_eks(
+    ctx: Context,
+    stack_name: Optional[str] = None,
+    install_agent: Optional[bool] = False,
```

Comment:
Not sure why we don't install the Agent by default?

Reply:
Code updated.